manim/mobject/svg/brace.py

```diff
@@ -228,3 +228,80 @@ def __init__(
             line_vector = np.array(point_2) - np.array(point_1)
             direction = np.array([line_vector[1], -line_vector[0], 0])
         Brace.__init__(self, Line(point_1, point_2), direction=direction, **kwargs)
+
+
+class ArcBrace(Brace):
```

Comment:
It seems like you've created a file for the graphical test, but it's actually not being tested. You'd have to write the code for the test into `brace.py` under the `tests` directory.

Reply:
is this where I add the scene?
> manim\tests\test_graphical_units\test_brace.py

Reply:
Yup, you can follow the format there. Also see [the docs](https://manimce--1548.org.readthedocs.build/en/1548/contributing/testing.html) if you'd like more info

Reply:
or do i create a new file?

Reply:
Since `ArcBrace` inherits from `Brace`, it makes sense that it falls into ``test_brace.py`

Reply:
I think I got it, I reread the docs